lib/pbench/test/unit/server/query_apis/test_datasets_detail.py

```diff
@@ -55,20 +75,25 @@ def test_missing_keys(self, client, server_config, keys, user_ok):
         however, Pbench will silently ignore any additional keys that are
         specified.
```

Comment:
The code doesn't test this assertion.  (The change made to `TestControllersList:test_missing_keys()` should be made here, too.)